shared/entity/type.go

```diff
@@ -70,6 +70,15 @@ const (
 
 	// TypeStorageBucket represents storage bucket resources.
 	TypeStorageBucket Type = "storage_bucket"
+
+	// TypeServer represents the top level /1.0 resource.
+	TypeServer Type = "server"
```

Comment:
What ordering, if any, is used with this list of constants, it seems rather arbitrary?

Reply:
None at all! Maybe we have separate `const` blocks for server level and project level resources, then order them alphabetically?

Reply:
yeah grouping them logically, and then alphabetically within those groups makes sense